elasticsearch/api.go

```diff
@@ -25,11 +25,47 @@ import (
 )
 
 var (
-	writeRequestIndex = "kubedb-system"
-	writeRequestID    = "info"
-	writeRequestType  = "_doc"
+	writeRequestIndex    = "kubedb-system"
+	writeRequestID       = "info"
+	writeRequestType     = "_doc"
+	CustomRoleName       = "readWriteAnyDatabase"
+	PrivilegeRead        = "read"
+	PrivilegeWrite       = "write"
+	PrivilegeCreateIndex = "create_index"
+	PrivilegeIndexAny    = "*"
+	ApplicationKibana    = "kibana-.kibana"
+	CreateSnapshot       = "create_snapshot"
+	Manage               = "manage"
+	ManageILM            = "manage_ilm"
+	ManageRoleup         = "manage_rollup"
+	Monitor              = "monitor"
+	ManageCCR            = "manage_ccr"
```

Comment:
the privilege constants should be`PrivilegeMonitor`, `PrivilegeManageCCR` etc.

Reply:
Why this block is `var` ? not `const` ?

Reply:
Also, make a seperate const block for the privileges. We shouldn't mix up things.